src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -984,9 +1042,14 @@ export default class DataTable extends Vue {
       columnHeadCell,
       columnHeadSortButton;
 
+    if ((element && element.classList.contains(INFO_ICON_CLASS)) || element.querySelector(`.${INFO_ICON_CLASS}`)) {
+      e.preventDefault();
+      return false;
+    }
+
     if (element) {
       while (element && !element.classList.contains(DATA_TABLE_CLASSES.HEAD)) {
-        if (element.nodeName === 'BUTTON') {
+        if (element.classList.contains(DATA_TABLE_CLASSES.SORTING)) {
```

Comment:
This breaks the blank space or label click functionality.

Reply:
Turns out it was the first conditional check that was the problem! this works fine as is 👍